src/inc/quic_datapath.h

```diff
@@ -125,12 +125,21 @@ PacketSizeFromUdpPayloadSize(
 // The top level datapath handle type.
 //
 typedef struct CXPLAT_DATAPATH CXPLAT_DATAPATH;
+typedef struct CXPLAT_DATAPATH_RAW CXPLAT_DATAPATH_RAW;
 
 //
 // Represents a UDP or TCP abstraction.
 //
 typedef struct CXPLAT_SOCKET CXPLAT_SOCKET;
 
+typedef enum CXPLAT_DATAPATH_TYPE {
```

Comment:
Does the QUIC layer _need_ to know the type? Again, with the goal of hiding all extra complexity from the upper layers, I think this should be defined and used internally only if possible.